src/app/components/BibPage/BibPage.jsx

```diff
@@ -80,8 +77,7 @@ const BibPage = (props) => {
                   shortenItems={shortenItems}
                   items={items}
                   bibId={bibId}
-                  title={`${bib.numItems} item${bib.numItems === 1 ? '' : 's'}
-                    associated with this record:`}
+                  title="AVAILABILITY"
```

Comment:
Do we have CSS rules for the title in <ItemHoldings>? It looks off now on the page. If no, no biggie then!